mono/utils/mono-context.h

```diff
@@ -18,6 +18,27 @@
 #include <signal.h>
 #endif
 
+#ifdef HOST_WIN32
+#include <windows.h>
+#endif
+
+#define MONO_CONTEXT_OFFSET(field, index, field_type) \
+    "i" (offsetof (MonoContext, field) + (index) * sizeof (field_type))
+
+/* Note [MonoContextSimdReg Alignment]:
+ *
+ * This type is aligned to allow direct movdqa/movaps.
+ */
+#if defined(__APPLE__)
```

Comment:
Ensuring all MonoContext instances will be aligned will be tricky as some are malloc'd or stack alloc'd by trampolines. If this is only needed for using movaps, we're better off just using movaps. The performance downside is not that bad.